pillar/ip_addrs.sls

```diff
@@ -1,2 +1,5 @@
 mine_functions:
-  network.ip_addrs: [eth0]
\ No newline at end of file
+  network.ip_addrs: [eth0]
+  hostname:
```

Comment:
To avoid confusion, we should call this "nodename", as there is also a "hostname" gain